### ISSUE TITLE
making curation-web more accessible

### ISSUE THREAD

**Issue by wowasa** [clarin-eric/curation-dashboard#249]
lighthouse shows a accessibility score of 69 - we should do something on accessibility

**Comment by wowasa** [comment]
adding alt attribute to the the img-tag of the clarin logo and a aria-label attribute to the enclosing a-tag increased the lighthouse accessibility score already to 95. The insufficient contrast ratio is remaining, but here we have to consider a general change in layout.
